src/python/pants/backend/python/dependency_inference/scripts/dependency_parser.py

```diff
@@ -114,7 +128,14 @@ def main(filename):
     # We have to be careful to set the encoding explicitly and write raw bytes ourselves.
     # See below for where we explicitly decode.
     buffer = sys.stdout if sys.version_info[0:2] == (2, 7) else sys.stdout.buffer
-    buffer.write("\n".join(sorted(visitor.imports)).encode("utf8"))
+    buffer.write(
+        json.dumps(
+            {
+                "imports": list(sorted(visitor.imports)),
+                "resources": list(sorted(visitor.resources)),
+            }
+        ).encode("utf8")
+    )
```

Comment:
This is a much better idea than what I've done in the `dockerfile_parser`.. will def steal this one :)